src/components/Onboarding/Screens/GardenMetadata.js

```diff
@@ -369,16 +302,115 @@ function GardenMetadata() {
+                assigned in the meantime.
+              </Help>
+            </div>
+            <div
+              css={`
+                display: flex;
+                flex-direction: column;
+                ${textStyle('body2')};
+                color: ${theme.contentSecondary};
+                margin-top: ${2 * GU}px;
+              `}
+            >
+              <span>Drag and drop or browse your files to upload one.</span>
+              <span
+                css={`
+                  color: ${formatValidationColor};
+                `}
+              >
+                Valid file formats are: JPG and PNG
+              </span>
+            </div>
+          </div>
+          <div
+            css={`
+              margin-top: ${2 * GU}px;
+              display: flex;
+              align-items: center;
+              justify-content: space-between;
+            `}
+          >
+            <div
+              css={`
+                min-width: 194px;
+              `}
+            >
+              <FileUploaderField
+                allowedMIMETypes={['image/jpeg', 'image/png']}
+                file={formData.logo_type}
+                onDragaAccepted={handleOnDragAccepted}
+                onDragRejected={handleOnDragRejected}
+                onFileUpdated={handleOnGardenLogoTypeUpdated}
+                label="GARDEN LOGO TYPE"
+              />
+            </div>
+            <div
+              css={`
+                min-width: 194px;
+              `}
+            >
+              <FileUploaderField
+                allowedMIMETypes={['image/jpeg', 'image/png']}
+                file={formData.logo}
+                onDragaAccepted={handleOnDragAccepted}
+                onDragRejected={handleOnDragRejected}
+                onFileUpdated={handleOnGardenLogoUpdated}
+                label="GARDEN LOGO"
+              />
+            </div>
+            <div
+              css={`
+                min-width: 194px;
+              `}
+            >
+              <FileUploaderField
+                allowedMIMETypes={['image/jpeg', 'image/png']}
+                file={formData.token_logo}
+                onDragaAccepted={handleOnDragAccepted}
+                onDragRejected={handleOnDragRejected}
+                onFileUpdated={handleOnTokenLogoUpdated}
+                label="TOKEN LOGO"
+              />
+            </div>
+          </div>
+        </div>
         <LinksBox
-          fieldTitle="Community Links"
+          fieldTitle="COMMUNITY LINKS"
           linksType={COMMUNITY_LINK_TYPE}
           links={formData.links.community}
           onAddLink={addLink}
           onRemoveLink={removeLink}
           onUpdateLink={updateLink}
         />
         <LinksBox
-          fieldTitle="Documentation Links"
+          fieldTitle="DOCUMENTATION LINKS"
           linksType={DOCUMENTATION_LINK_TYPE}
           links={formData.links.documentation}
           onAddLink={addLink}
```

Comment:
After trying it once again and see how several errors are displayed. We map them and create a `div` for each inside `Info`. What do you think of creating an `Info` component for each instead?

Reply:
in this case i don't think this is so good because it can generate a bunch of  divs with their correspondents paddings margins that would generate a lot of unnecesary spacing